omnibus/omnibus-test.sh

```diff
@@ -61,7 +61,7 @@ y/8SReCpC71R+Vl6d4+Dw6GFdL+6k6W558dPfq3UeV8HPWQEaM7/jXDUKJZ0tB6a
 -----END DH PARAMETERS-----
 " | sudo tee /etc/opscode/dhparam.pem
 
-sudo chef-server-ctl reconfigure || true
+sudo chef-server-ctl reconfigure --chef-license accept || true
```

Comment:
some cases we're doing `--chef-license accept` and some `--chef-license=accept`; could you make it consistently one way or the other?

Reply:
Here I would do `accept-no-persist` because I'm thinking this test runs on a CI environment. I would rather not leave license acceptance marker files around on CI nodes.

Reply:
Fixed